lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -507,7 +508,7 @@ def setUp(self):
         OverrideFieldData.provider_classes = None
         # pylint: disable=protected-access
         for block in iter_blocks(course):
-            block._field_data = OverrideFieldData.wrap(coach, block._field_data)
+            block._field_data = OverrideFieldData.wrap(coach, course.id, block._field_data)
             new_cache = {'tabs': [], 'discussion_topics': []}
```

Comment:
@cguardia  do we need to check here OverrideFieldData is wrap or not? if yes then cache grading policy?